database/couchDb/db.js

```diff
@@ -0,0 +1,52 @@
+require('dotenv').config();
+const axios = require('axios');
+const nano = require('nano')(process.env.COUCHDB);
+
+
+let start = Date.now();
+let total = 10000000;
+let batchStart = 1;
+let batchSize = 1500;
+let wait = 0;
+nano.db.destroy('sdc')
+  .then(() => nano.db.create('sdc'))
+  .then(async () => {
+    console.log('sdc database destroyed and created')
+    const db = nano.use('sdc')
+
+    while (batchStart < total) {
+      let docs = [];
+      for (let i = batchStart; i < batchStart + batchSize; i++) {
+        let minRate = Math.floor(Math.random() * 500) + 30;
+        let reservations = [];
+        let dateCounter = new Date();
+        let start = Math.floor(Math.random() * (10 - 1 - 0 + 1) + 0)
+        for (let k = 0; k < Math.floor(Math.random() * 30) + 1; k++) {
+          let end = start + Math.floor(Math.random() * 9) + 1;
+          reservations.push({
+            id: k,
+            startDate: new Date().setDate(dateCounter.getDate() + start),
```

Comment:
Looks like this is a separate seeding script than for PostGres. Did you look into using one script ? If yes, did run into any issues  with that approach ?

Reply:
I had already created both scripts when I saw that note in glearn, but they are basically the same data generation script, the structures were just slightly different.